scripts/evaluate_results/aggregrate_experiment.py

```
@@ -6,7 +6,15 @@
 def aggregate_results(
     log_dir: Path,
-    models=["LSTM", "Transformer", "GRU", "RNN", "LogisticRegression", "LGBMClassifier", "TCN"],
+    models=[
+        "LSTM",
+        "Transformer",
+        "GRU",
+        "RNN",
+        "LogisticRegression",
+        "LGBMClassifier",
+        "TCN",
+    ],
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)